app/js/reducers/labOrders/labConceptsReducer.js

```diff
@@ -5,15 +5,36 @@ import {
 } from '../../actions/actionTypes';
 import initialState from '../initialState';
 
+
+const removeDuplicateTests = (tests) => {
+  const uniqueTestIds = Array.from(new Set(tests.map(test => test.uuid)));
+  const uniqueTests = []; 
+  tests.forEach((test) => {
+    const testIndex = uniqueTestIds.indexOf(test.uuid);
+    if (testIndex !== -1) {
+      uniqueTests.push(test);
+      uniqueTestIds.splice(testIndex, 1);
+    }
+  });
+  return uniqueTests;
+};
+
 export default (state = initialState.labConcepts, action) => {
   switch (action.type) {
-    case FETCH_LAB_CONCEPTS_SUCCESS:
+    case FETCH_LAB_CONCEPTS_SUCCESS: {
+      const concepts = [...action.payload.data.setMembers];
+      const panels = concepts.filter(concept => concept.set);
+      const panelTests = [].concat(...panels.map(panel => panel.setMembers));
```

Comment:
you don't need to use `.concat`, just map, destructure and use cos the map still returns an array anyway and what you're joining it to is just an empty array

Reply:
Hi @rhenshaw56 
The result of the map is actually an Array of Array, something like this `[[...], [...]]`, I needed the `.concat` to flatten out the nested Array.

Reply:
`const [ panelSet ] = panels.map(panel => panel.setMembers` don't work ?

Reply:
Yes, it does not work.

Reply:
alright then!